editors/vscode/src/package-manager.ts

```diff
@@ -0,0 +1,209 @@
+import { window, workspace } from 'vscode';
+import * as fs from 'fs';
+
+// error message
+export const dataDirErrorMessage = 'Can not find package directory.';
+
+// https://github.com/typst/packages#local-packages
+export function getDataDir() {
```

Comment:
The `process.env` uses looks not great to me. I would like to use same configuration as that from LSP, which already computes the directory to local packages.

Could you add a resource that provides data directory at the LSP side? 

Specifically, we can add a resource `dirs/local-package` here, which returns same value as `function getLocalPackagesDir` in this file:

https://github.com/Myriad-Dreamin/tinymist/blob/96d5ba0c0075072b4e602dd6291b8f1d1c2de0be/crates/tinymist/src/server.rs#L266-L269

Then, we can get the resources by `tinymist.getResource`:

https://github.com/Myriad-Dreamin/tinymist/blob/96d5ba0c0075072b4e602dd6291b8f1d1c2de0be/editors/vscode/src/lsp.ts#L18-L21

https://github.com/Myriad-Dreamin/tinymist/blob/96d5ba0c0075072b4e602dd6291b8f1d1c2de0be/editors/vscode/src/lsp.ts#L32-L34